crowbar_framework/app/models/ceph_service.rb

```diff
@@ -252,7 +252,7 @@ def validate_proposal_after_save proposal
     end
 
     unless radosgw_nodes.empty?
-      ProposalObject.find_proposals("swift").each {|p|
+      Proposal.where(barclamp: "swift").each {|p|
```

Comment:
The part that confuses me is that you used `.all` in https://github.com/crowbar/barclamp-swift/pull/286/files#diff-01dcdaa5e35c14cdff38ef4940cdb8fdR308

So trying to understand which way is better :-)